### DISCUSSION TITLE
Time proof

### DISCUSSION THREAD

**Discussion by haidar632** [Chia-Network/chia-blockchain#6134] [Support]
Hii guys when my plots pass to challange my peroid time is increase to 9 secound and when not passad my time 0.00100 sec, what can i do?

**Comment by haidar632** [comment]
![Uploading Screenshot_20210528-003329_TeamViewer.jpg…]()

**Comment by BrandtH22** [comment]
@haidar632 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - only 1/512 plots are expected to pass the current filter and of those only a small portion will actual have a valid proof for the challenge. So long as your proof time responses are under 30 seconds your proofs are still eligible for farming blocks.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
